terraform/dev/main.tf

```diff
@@ -106,6 +106,8 @@ module "data_share_service" {
 
   admin_alerts_enabled           = false
   database_tunnel_alerts_enabled = false
+
+  admin_login_allowed_ip_blocks = null
```

Comment:
can we not just provide a default and not set it here?

Reply:
I'd like not restricting to be a conscious choice so we don't forget it in prod